package.json

```diff
@@ -10,6 +10,7 @@
     "prop-types": "^15.7.2",
     "react": "^17.0.1",
     "react-dom": "^17.0.1",
+    "react-icons": "^3.11.0",
```

Comment:
I love how you have done research on implementing new libraries and tools to make our application more professional.  Thanks for going out of your way to get these into our project.

Reply:
Yeah this is awesome! I will definitely have to look at the documentation for this.

There are so many packages for pre-built components that can be so useful in the future!